### PR TITLE
Added a datePicker, made new expense show in the list and minor changes 

### PR DESCRIPTION
**Pull request by Ololopololo**
Made calendar an image again, moved it to the left, and changed back and done buttons color to darkblue

### CODE REVIEW

File: Outlay/Podfile.lock [hunk 1]
```diff
@@ -17,4 +17,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: d45de82dcf86308ce89849c9103d3ec82aca5a74
 
-COCOAPODS: 1.11.2
```
Comment (L1keMagic) [inline Outlay/Podfile.lock:20]: could you please remove this

File: Outlay/Outlay/NewExpense/NewExpenseViewController.swift [hunk 2]
```diff
@@ -123,16 +132,30 @@ extension NewExpenseViewController {
...
+        return imageView
+    }
+    // MARK: - Create Date Picker
+    private func createDatePicker() {
+        datePicker.datePickerMode = .date
+        datePicker.addTarget(self, action: #selector(dateChange(datePicker:)), for: UIControl.Event.valueChanged)
+        if #available(iOS 13.4, *) {
+            datePicker.preferredDatePickerStyle = .compact
+        } else {
+            // Fallback on earlier versions
```
Comment (L1keMagic) [inline Outlay/Outlay/NewExpense/NewExpenseViewController.swift:156]: could you please clarify what the logic will be executed it there is less iOS version used

File: Outlay/Outlay/NewExpense/NewExpenseViewController.swift [hunk 3]
```diff
@@ -34,9 +36,14 @@ class NewExpenseViewController: UIViewController {
     @objc func backButton() {
         self.dismiss(animated: true)
     }
-    // MARK: - Add action for calendar
-    @objc func openCalendar() {
-        Logger.information(message: "open calendar button touched")
+    @objc func dateChange(datePicker: UIDatePicker) {
```
Comment (L1keMagic) [inline Outlay/Outlay/NewExpense/NewExpenseViewController.swift:39]: could you please change method name to calendarDateChanged()

Comment (L1keMagic) [reply Outlay/Outlay/NewExpense/NewExpenseViewController.swift:39]: Also you can give another method name which fits better here

**Review by L1keMagic — CHANGES_REQUESTED** [review]

File: Outlay/Outlay/NewExpense/NewExpenseViewController.swift (hunk 2 above)
Comment (Ololopololo) [reply Outlay/Outlay/NewExpense/NewExpenseViewController.swift:156]: Actually we don't need it at all, so i'll just remove it.

File: Outlay/Outlay/NewExpense/NewExpenseViewController.swift (hunk 3 above)
Comment (Ololopololo) [reply Outlay/Outlay/NewExpense/NewExpenseViewController.swift:39]: Done

File: Outlay/Outlay/NewExpense/NewExpenseViewController.swift [hunk 4]
```diff
@@ -34,9 +36,14 @@ class NewExpenseViewController: UIViewController {
     @objc func backButton() {
         self.dismiss(animated: true)
     }
-    // MARK: - Add action for calendar
-    @objc func openCalendar() {
-        Logger.information(message: "open calendar button touched")
+    @objc func didExpenceDateCnahged(datePicker: UIDatePicker) {
```
Comment (L1keMagic) [inline Outlay/Outlay/NewExpense/NewExpenseViewController.swift:39]: could you please spell "expen**s**e" word correctly

**Review by L1keMagic — CHANGES_REQUESTED** [review]

File: Outlay/Outlay/NewExpense/NewExpenseViewController.swift (hunk 4 above)
Comment (Ololopololo) [reply Outlay/Outlay/NewExpense/NewExpenseViewController.swift:39]: fixed

Comment (L1keMagic) [reply Outlay/Outlay/NewExpense/NewExpenseViewController.swift:39]: I dont see any changes

**Comment by L1keMagic** [comment]
approved

**Review by L1keMagic — APPROVED** [review]
approved

**Pull request merged**